src/App.tsx

```diff
@@ -1,26 +1,141 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, getTodos, deleteTodo, createTodo } from './api/todos';
+import { Errors } from './components/errors/errors';
+import { Footer } from './components/footer/footer';
+import { ToDoList } from './components/todoList/todoList';
+import { Header } from './components/header/header';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoSatus';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState('');
+  const [idTodo, setIdTodo] = useState(0);
+  const [newTodo, setNewTodo] = useState({
+    userId: USER_ID,
+    title: '',
+    completed: false,
+  });
+
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => {
+        switch (status) {
+          case TodoStatus.Active:
+            setTodos(todosFromServer.filter(todo => !todo.completed));
+            break;
+          case TodoStatus.Completed:
+            setTodos(todosFromServer.filter(todo => todo.completed));
+            break;
+
+          default:
+            setTodos(todosFromServer);
```

Comment:
It would be better not to do this here. Create a separate function, in file utils.ts. And create variable filteredTodos = call your function.